types/numjs/index.d.ts

```diff
@@ -0,0 +1,589 @@
+// Type definitions for numjs 0.14
+// Project: https://github.com/nicolaspanel/numjs#readme
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export as namespace nj;
+import * as BaseNdArray from 'ndarray';
+
+export type NdType<T> = BaseNdArray.dtType | BaseNdArray.Data<T>;
+
+export interface NdArray<T = number> extends BaseNdArray<T> {
+	ndim: number;
+	T: NdArray<T>;
+	slice(...args: number[]): NdArray<T>;
+
+	/**
+	 * Return a copy of the array collapsed into one dimension using row-major order (C-style)
+	 *
+	 * @returns {NdArray}
```

Comment:
Don't include a JSDoc annotation that repeats information already in the type definition.